spec/models/user_spec.rb

```diff
@@ -3,5 +3,69 @@
 require "rails_helper"
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  context "user attrs validation" do
+    let(:user_attributes) do
+      {
+        email: email,
+        password: password,
+        password_confirmation: password_confirmation
+      }
+    end
+    let(:email) { "valid_email@gmail.com" }
+    let(:password) { "123456" }
+    let(:password_confirmation) { password }
+    let(:user) { described_class.new(user_attributes) }
+    it "accepts valid params" do
+      expect(user.valid?).to be_truthy
+    end
+    context "role of the created user" do
+      it "accepts false" do
```

Comment:
`it "accepts false"` 
don't think it describes the behaviour
better write `'properly validates params'` 
pls check all places